wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -0,0 +1,176 @@
+import os
+
+import tensorflow as tf
+
+from transformers import (
+    BertConfig,
+    BertTokenizer,
+    TFBertForSequenceClassification,
+    glue_convert_examples_to_features,
+    glue_processors,
+)
+
```

Comment:
why the spaces in between the external libraries?

Reply:
This was fixed in the new commits.